### PR TITLE
Creating detail View with template to show shortstories by given id, …

### PR DESCRIPTION
**Pull request by maryamyasaei**
…also writing some unit tests

### CODE REVIEW

File: shortstories/urls.py [hunk 1]
```diff
@@ -2,5 +2,7 @@
 from . import views
 
 urlpatterns = [
-    path('', views.index, name='index')
+    path('', views.index, name='index'),
+    path('<int:shortstory_id>/', views.detail, name='detail')
+
 ]
```
Comment (hesamd) [inline shortstories/urls.py]: new line at the end of file.

File: shortstories/tests.py [hunk 2]
```diff
@@ -15,10 +15,25 @@ def  test_no_shortstories(self):
...
+    def  test_no_shortstories(self):
+        user_id = User.objects.create(id=1)
+        current_shortstories = create_shortstories(author= user_id, status = 'p', title = "Villete", publication_date ='2000-08-25')
+        response = self.client.get(reverse('shortstories:detail', args=(current_shortstories.id,)))
+        self.assertEqual(response.status_code, 404)
+
+class shortstoriesDetailViewTests(TestCase):
+    def  test_current_shortstories(self):
+        user_id = User.objects.create(id=1)
+        current_shortstories = create_shortstories(author= user_id, status = 'p', title = "Villete", publication_date ='2000-08-25')
```
Comment (hesamd) [inline shortstories/tests.py]: these are all one short story hence you need to use a proper variable name..i.e. `current_shortstory`; this applies to all of your tests

File: shortstories/tests.py [hunk 3]
```diff
@@ -15,10 +15,25 @@ def  test_no_shortstories(self):
...
         response = self.client.get(url)
         self.assertContains(response, current_shortstories.title)
+
+
+class shortstoriesDetailViewTests(TestCase):
+    def  test_no_shortstories(self):
+        user_id = User.objects.create(id=1)
+        current_shortstories = create_shortstories(author= user_id, status = 'p', title = "Villete", publication_date ='2000-08-25')
+        response = self.client.get(reverse('shortstories:detail', args=(current_shortstories.id,)))
+        self.assertEqual(response.status_code, 404)
```
Comment (hesamd) [inline shortstories/tests.py]: I don't understand why you're still getting 404 here. Don't you create a short story and try to see the same one? why are you still getting 404 then? You should get 200.

**Review by hesamd — CHANGES_REQUESTED** [review]

File: shortstories/tests.py (hunk 3 above)
Comment (maryamyasaei) [reply shortstories/tests.py]: I used it is for a test which checks  there would be a 404 error, if there is no short story with that given id, still 200?

**Review by hesamd — APPROVED** [review]

**Pull request merged**